_docs/setup/install-kubernetes.md

```diff
@@ -36,21 +37,19 @@ Quick Start instructions to install and configure Istio in a Kubernetes cluster.
  oc adm policy add-scc-to-user anyuid -z istio-ingress-service-account -n istio-system
  oc adm policy add-scc-to-user anyuid -z istio-egress-service-account -n istio-system
   ```  
-You can expose services either via istio ingress or using default openshift router.
+You can expose services either via Istio ingress or using default openshift router.
```

Comment:
what's that openshift thing btw ?

Reply:
We should ask Rajagopalan, added a link to this
https://docs.openshift.com/container-platform/3.3/dev_guide/getting_traffic_into_cluster.html#using-a-router